Sources/Apodini/Semantic Model Builder/Model/Endpoint.swift

```diff
@@ -158,104 +164,50 @@ class EndpointsTreeNode {
     }
     
     func addEndpoint<H: Handler>(_ endpoint: inout Endpoint<H>, at paths: [PathComponent]) {
-        if paths.isEmpty {
+        guard !paths.isEmpty else {
             // swiftlint:disable:next force_unwrapping
```

Comment:
Misplaced SwiftLint comment, previously addressing the force_unwrap in the precondition error message